build/Makefile.kits

```diff
@@ -208,10 +260,14 @@ ifneq ($(COMBINED_PACKAGES),1)
   ifeq ($(PF),Linux)
     ifneq ($(PF_ARCH),ppc)
 	# Copy omi kit depending on openssl version
-      ifeq ($(IS_OPENSSL_100),1)
-	cp `find $(OMIKITS_DIR) -name omi-*ssl_100.ulinux.$(PF_ARCH).$(PACKAGE_SUFFIX)` $(INTERMEDIATE_DIR)/
+      ifeq ($(IS_OPENSSL_110),1)
+        cp `find $(OMIKITS_DIR) -name omi-*ssl_110.ulinux.$(PF_ARCH).$(PACKAGE_SUFFIX)` $(INTERMEDIATE_DIR)/
       else
-	cp `find $(OMIKITS_DIR) -name omi-*ssl_098.ulinux.$(PF_ARCH).$(PACKAGE_SUFFIX)` $(INTERMEDIATE_DIR)/
+        ifeq ($(IS_OPENSSL_100),1)
+	  cp `find $(OMIKITS_DIR) -name omi-*ssl_100.ulinux.$(PF_ARCH).$(PACKAGE_SUFFIX)` $(INTERMEDIATE_DIR)/
+        else
+	  cp `find $(OMIKITS_DIR) -name omi-*ssl_098.ulinux.$(PF_ARCH).$(PACKAGE_SUFFIX)` $(INTERMEDIATE_DIR)/
+        endif
```

Comment:
I'm not really sure here. But I wonder if we can just copy any Linux kits, since these are universal builds anyway, with a single `cp` line like:

```
cp `find $(OMIKITS_DIR) -name omi-*ssl_*.ulinux.$(PF_ARCH).$(PACKAGE_SUFFIX)` $(INTERMEDIATE_DIR)/
```

Again, not 100% certain, but I think that would work. I'm thinking about SSL 0.9.8 and, perhaps in my lifetime, we drop support for that. This sort of change here would make that work slightly easier.

Reply:
It should work but by making it explicit we are ensuring correct package are getting copied. Suppose in future we have dropped 0.9.8 and by mistake someone copied 0.9.8 package under omi-kit, with current code it will not package it in shell bundle. But with generic code you have written will bundle it even though it is not needed. I would prefer not to change the current model. Let me know if you have concerns.

Reply:
I'm fine with this, no problem.